factory/cryptoComponents.go

```diff
@@ -167,6 +174,19 @@ func (ccf *cryptoComponentsFactory) Create() (*cryptoComponents, error) {
 		return nil, err
 	}
 
+	blsKeyGenerator := signing.NewKeyGenerator(mcl.NewSuiteBLS12())
```

Comment:
shouldn't it be created based on a config? similar to `factory/cryptoComponents.getSuite()`